luci/luci-app-ffwizard-falter/luasrc/model/cbi/freifunk/assistent/wireless.lua

```diff
@@ -267,10 +267,20 @@ function main.write(self, section, value)
     prenetconfig.ipaddr=dhcpmeshnet:minhost():string()
     prenetconfig.netmask=dhcpmeshnet:mask():string()
     prenetconfig.ip6assign="64"
-    prenetconfig.type="bridge"
+    -- prenetconfig.type="bridge" -- as of 21.02 type=bridge is not used for interface sections
```

Comment:
Is there a specific reason for keeping that line?

Reply:
So that people can read the comment behind it and know why it's been changed.